src/components/Home.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { NavLink } from 'react-router-dom';
+import Details from './Details';
+
+const Home = () => {
+  const data = useSelector((state) => state.home.data);
+
+  return (
+    <ul className="list-group">
+      {data.map((item) => (
+        <NavLink key={item.year} to={`/details/${item.date}`}>
+          <Details year={item.year} />
+        </NavLink>
+      ))}
+    </ul>
+  );
+};
```

Comment:
- [ ] While you are expected to personalize your content, you are still expected to follow the design provided. Your Homepage is missing the descriptive banner section provided in the design - in the image, the section containing **EUROPE 6958 VIEWS**. Kindly add it.
- [ ] According to the design given, the pattern for the list of years on the Homepage should follow a grid pattern and also have the alternating (light and thick) colors as highlighted. Kindly add this.

  Your Design | Behance Design
  :---:|:---:
  ![vechin-own-home](https://user-images.githubusercontent.com/36542986/152467528-33aa3ca8-a112-4d0a-a9bf-c84a9744df4f.jpg) | ![vechin-orig-home](https://user-images.githubusercontent.com/36542986/152467524-87fca799-0603-4d40-b207-df629e752289.jpg)

- [ ] Your Homepage should also have some kind of filtering mechanism (eg: a search functionality) by which users can filter contents on the page. Kindly add this functionality.

  ![vechin-search](https://user-images.githubusercontent.com/36542986/152468089-047c9ebd-ea88-49b0-80cd-0079faf50752.jpg)